src/diy/cases.md

```diff
@@ -105,7 +105,18 @@ The SlimeVR community has built a huge number of cases for style, internals and
 
 [Github](https://github.com/frosty6742/frozen-slimes)
 
+## Frozen Slimes V2
+*Design by artemis/frosty*
 
+<img class="caseImage"
+     src="https://user-images.githubusercontent.com/98719680/218297952-111f2c94-485c-4ddd-be9c-3e330d8ab26a.png"
+     alt="PCB"
+     width="600" height="400">
+     
+* Support mpu6050s, bmis160s, and BNO085s all on one pcb!
```

Comment:
The IMU names should be written the way they are written on the respective websites, i.e. MPU-6050 and BMI160.